src/view/main-mobile.js

```diff
@@ -183,11 +183,16 @@ const MainStack = createStackNavigator(
     NewAddress,
     Password,
     LoaderSyncing,
+    Channels,
```

Comment:
why remove the modal transition for channel/notification lists?

Reply:
This modal transition was only a workaround the empty list issue. For some reason list items were rendering for modals and not slide transitions.